domains/transfer/Shared/DataContext/Migrations/20241107113909_AddTransferAgreementType.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddTransferAgreementType : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Type",
+                table: "TransferAgreements",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<int>(
+                name: "Type",
+                table: "TransferAgreementProposals",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
```

Comment:
I useally just delete the down part :D

Reply:
Should we handle this in general for all migrations?